frontend/src/components/panels/project_list_panel/ProjectListPanel.js

```diff
@@ -23,8 +25,11 @@ class ProjectListPanel extends React.Component {
         meep_service.getProjectDetailsById(project_id)
             .then(selectProject)
             .then(this.props.dispatch)
-    }
+    };
+
     render() {
+        console.log(this.props.filters)
```

Comment:
Looks like there's some console.log statements still here, haha. Can we also clean this up part?

Reply:
I forgot to push that commit...somehow. Should be fixed now. Thanks.